examples/passport-oauth2-login/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "passport-oauth2-login",
```

Comment:
The name should be something like `@loopback/example-passport-oauth2`.

Reply:
agreed